staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1481,6 +1485,8 @@ message Image {
     // It must only be treated as a recommendation -- the client can still request that the image be deleted,
     // and the runtime must oblige.
     bool pinned = 8;
+    // Mountpoint is an absolute path where the OCI object is mounted on disk.
+    string mountpoint = 9;
```

Comment:
same.. image volume mount type here so one can discern how the image may be mounted with a volume mount request..

Reply:
Yep, fixed.